.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and deploy WCAG Primer
+
+on:
+- push
+
+jobs:
+  # A really basic check to see if this builds okay.
+  only-build:
+    runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/main' || github.ref != 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+    - run: bundle install
+    - run: bundle exec middleman build --clean
+
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+    - run: bundle install
```

Comment:
The docs for ruby/setup-ruby seem to [suggest that `bundler-cache: true` will be doing `bundle install` for us already](https://github.com/ruby/setup-ruby#single-job).

Reply:
That's good to know - ta! I've updated to remove `bundle install`.